sys/include/phydat.h

```diff
@@ -87,16 +87,16 @@ enum {
     UNIT_M2,        /**< square meters */
     UNIT_M3,        /**< cubic meters */
     /* kinetic */
-    UNIT_G,         /**< gravitational force */
+    UNIT_G_FORCE,   /**< gravitational force equivalent */
```

Comment:
```suggestion
    UNIT_G_FORCE,   /**< gravitational force equivalent */
    UNIT_G = UNIT_G_FORCE, /**< deprecated, use UNIT_G_FORCE instead */
```
This should hopefully work with c2rust too